docs/QUnit/module.md

```diff
@@ -61,8 +61,6 @@ The module's callback is invoked with the test environment as its `this` context
 
 When you are debugging your code, you will often need to _only_ run a subset of tests. You can append `only` to `QUnit.module` to specify which module you want to run.
```

Comment:
To make this a bit clearer that it supports more than one module, maybe this line could be updated to say: `...specify which module(s) you want to run.`

Reply:
Done. Thanks :-).